cddl/accepted-claims-set.cddl

```diff
@@ -0,0 +1,6 @@
+accepted-claims-set = {
+  &(evidence-triples: 0) => [ + reference-triple-record ]
+  ? &(identity-triples: 2) => [ + identity-triple-record ]
+  ? &(coswid-triples: 6) => [ + ev-coswid-triple-record ]
+  * $$triples-map-extension
```

Comment:
conventionally this should be called `accepted-claims-set-extension`

Reply:
ok